src/StartMenu.ts

```diff
@@ -296,64 +299,86 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const plugin = getPlugin(EXTENSION_POINT_ID, EXTENSION_ID);
```

Comment:
if you are requesting a specific extension with a specific id, why using an extension mechanism at all?